bindings/guide.md

```diff
@@ -37,6 +37,7 @@ Basic functions:
     - Associations
     - Topic Models
     - Topic Distributions
+    - Deepnets
```

Comment:
In my opinion, this item should be the last one in order to follow the chronological order.